commercetools/commercetools-sdk-java-api/src/main/java/com/commercetools/api/models/type/TypeMixin.java

```diff
@@ -1,10 +1,18 @@
 
 package com.commercetools.api.models.type;
 
+import java.util.List;
+
 import com.commercetools.api.models.Referencable;
 import com.commercetools.api.models.ResourceIdentifiable;
 
 public interface TypeMixin extends Referencable<Type>, ResourceIdentifiable<Type> {
+    public List<FieldDefinition> getFieldDefinitions();
+
+    default FieldDefinition getFieldDefinitionByName(final String name) {
+        return getFieldDefinitions().stream().filter(def -> def.getName().equals(name)).findFirst().orElse(null);
+    }
+
     @Override
     public default TypeResourceIdentifier toResourceIdentifier() {
```

Comment:
Here i violated my own rule 🤦. Now thinking about how to deal with it. Would like to avoid that the mixin interfaces can be seen as a complete model.